internal/ssm/install.go

```diff
@@ -18,37 +21,104 @@ import (
 	"github.com/aws/eks-hybrid/internal/util/cmd"
 )
 
-const (
-	installerPath = "/opt/ssm/ssm-setup-cli"
-	configRoot    = "/etc/amazon"
+// TODO: a better way to override this
+var (
```

Comment:
i think this can go back to a const?
```suggestion
const (
```